cmd/put.go

```diff
@@ -352,13 +352,13 @@ func parsePutFile(path, meta, requester string, splitter bufio.SplitFunc, base64
 }
 
 func parseMetaString(meta string) map[string]string {
-	if meta == "" {
-		return nil
-	}
-
 	kvs := strings.Split(meta, ";")
 	mm := make(map[string]string, len(kvs))
```

Comment:
Just a note, but you don't need to supply a length to a map, and on maps this small and short lived, I wouldn't bother. Feel free to leave this as it is, though.